tests/methods/embedding-3/input.dat

```diff
@@ -1,7 +1,8 @@
 import forte
 
 refscf =   -115.6554016101969893
-refdsrgpt2 = -114.704305626419853
+#refdsrgpt2 = -114.704305626419853
```

Comment:
Remove comment here and the bit `# New value, should be lower than SCF energy`

Reply:
Removed